server/reportmanager/models.py

```diff
@@ -523,9 +524,11 @@ def get_report(self):
                 reported_at=self.reported_at,
                 uuid=self.uuid,
                 url=urlsplit(self.url),
-                breakage_category=self.breakage_category.value
-                if self.breakage_category is not None
-                else None,
+                breakage_category=(
+                    self.breakage_category.value
+                    if self.breakage_category is not None
+                    else None
+                ),
```

Comment:
This change is unrelated, and `ruff` doesn't care either way. But my editor keeps insisting that this is the right way to do it, and I tend to agree - it's a bit confusing otherwise. Happy to revert, though, if we wanna keep commits super clean. :)